security_mvp/src/models/dangerous_object_detector.py

```
@@ -0,0 +1,18 @@
+from __future__ import annotations
+
+from models.interfaces import DetectionResult
+
+
+class DangerousObjectDetector:
+    def __init__(self) -> None:
+        self.model_version = "stub"
+
+    def detect(self, frame) -> DetectionResult:
```

Ruff (0.14.13)

Unused method argument: `frame`

(ARG002)